conf/channels/Realtime_HL7V2_Processor.xml

```diff
@@ -550,7 +551,7 @@ channel.basicPublish($(&apos;AMQP_EXCHANGE&apos;),
           <serializationProperties class="com.mirth.connect.plugins.datatypes.hl7v2.HL7v2SerializationProperties" version="3.4.2">
             <handleRepetitions>true</handleRepetitions>
             <handleSubcomponents>true</handleSubcomponents>
-            <useStrictParser>false</useStrictParser>
+            <useStrictParser>true</useStrictParser>
```

Comment:
Is this part of the queue changeover, or the XML format change? Or something else?

Reply:
Oh, I see you comment now, I didn't read the whole thing before, sorry